src/encoded/types/base.py

```diff
@@ -63,6 +67,11 @@ def _award_viewing_group(award_uuid, root):
     (Allow, 'group.submitter', ['add']),
 ]
 
+# New status must contain current status in list to be valid transition.
+STATUS_TRANSITION_TABLE = {
+    'released': ['in progress'],
```

Comment:
So this is "TO": [ "FROM1", "FROM2"...]??

Reply:
Yes, if you are trying to go to released you must be in progress.